tests/doc-links.spec.ts

```diff
@@ -58,7 +58,9 @@ test("Ensure the documentation link text and link targets are present: Server Se
   await openServerSetting(page, "core.storage_buckets_address");
   await validateLink(
     page,
-    "How to manage storage buckets and keys",
+    lxdVersion === "5.21-edge"
```

Comment:
Lets not have this, but rather wait for that change to be backported.

Similar to https://github.com/canonical/lxd-ci/pull/525#issuecomment-3096103753

This way once you've got the UI tests up and running in lxd-ci repo, we can see if there are missing expected backports. Otherwise this is just going to break again in 5.21 when its backported.

Reply:
updating this in https://github.com/canonical/lxd-ui/pull/1400